nestkernel/connector_model.h

```diff
@@ -115,57 +135,59 @@ class ConnectorModel
   bool
   is_primary() const
   {
-    return is_primary_;
+    return has_property( properties_, ConnectionModelProperties::IS_PRIMARY );
```

Comment:
I would change this and all other corresponding code to
```suggestion
    return has_property( ConnectionModelProperties::IS_PRIMARY );
```
where `has_property()` is renamed from `has_flag_set()`, see below.